File: Calculator.java
import java.util.Scanner;
public class Calculator {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);

    System.out.print("Enter Number ans operation to perform: ");
    int firstNum = input.nextInt();
    String operator = input.next();
    int secondNum = input.nextInt();
    
    if(operator.equals("+")){
        int sum = firstNum + secondNum;
        System.out.println("sum " +sum);
    }else if(operator.equals("*")){
        int product = firstNum * secondNum;
        System.out.println("product: " + product);
    }else if(operator.equals("/")){
        double quotient = firstNum / secondNum;
        System.out.println("product: " + quotient);
    }
    
    }
}

File: chapter 2/Calculator.py
first_number = int(input("Enter First Number: "))
operator = input("Enter operator: ")
second_number = int(input("Enter second Number: "))

if operator == '+':
    sum_of_numbers = first_number + second_number
    print(f" sum of {first_number} is {sum_of_numbers}")
elif operator == '-':
    sum_of_numbers = first_number - second_number
    print(f" sum of {first_number} is {sum_of_numbers}")
elif operator == '/':
    sum_of_numbers = first_number / second_number
    print(f" sum of {first_number} is {sum_of_numbers}")
else:
    print("Enter valid number")
